web/src/components/PoisDesktop.tsx

```diff
@@ -141,8 +138,8 @@ const PoisDesktop = ({
             <ListHeader>{t('listTitle')}</ListHeader>
           )}
 
-          {currentPoi && currentPoiFeature ? (
-            <PoiDetails poi={currentPoi} feature={currentPoiFeature} direction={direction} t={t} />
+          {currentPoi ? (
+            <PoiDetails poi={currentPoi} feature={currentPoi.feature} direction={direction} t={t} />
```

Comment:
@sarahsporck is this the same? Or do we have to select the current feature from `currentFeatureOnMap`?